{
  this(ErrorType.ERROR_INTERNAL_EXCEPTION,message,null);
}
